dotnet/src/SemanticKernel/SemanticKernel.csproj

```diff
@@ -39,9 +39,37 @@
     <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
       <_Parameter1>SemanticKernel.UnitTests</_Parameter1>
     </AssemblyAttribute>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>Microsoft.SemanticKernel.Connectors.OpenAI</_Parameter1>
```

Comment:
For a future update: I don't think we should need to expose these internals. Looks like this is needed to access `DelegateTypes` and `SKFunction`. Can/should we implement a new ISKFunction class in the OpenAPI project instead?